gpkit/tests/t_sub.py

```diff
@@ -112,10 +112,20 @@ def test_signomial(self):
             self.assertTrue(isinstance(subbed, Signomial))
             self.assertEqual(subbed, 2*x - y - D)
 
+    def test_partial_sub_signomial(self):
+        """Test SP partial x0 initialization"""
+        x = Variable('x')
+        y = Variable('y')
+        with SignomialsEnabled():
+            m = Model(x, [x + y >= 1, y <= 0.5])
+        m.localsolve(x0={x: 0.5}, verbosity=0)
+        self.assertEqual(m.program.gps[0].constraints[0].exp[x], -1./3)
```

Comment:
Great to see the test.

not convinced that this test belongs in `t_sub.py` (which is meant for testing `substitution` and `sub` methods -- not initial guesses)

I'd put this in the `TestSP` class in `t_model.py`.